### ISSUE TITLE
scylla-jmx release no deb package

### ISSUE THREAD

**Issue by owanio1992** [scylladb/scylla-jmx#247]
I read this doc 
https://opensource.docs.scylladb.com/stable/getting-started/install-scylla/install-jmx.html

I not fond .deb in release page 

![Image](https://github.com/user-attachments/assets/e847f3d9-349b-4b6c-91f5-49484ecbaf93)

**Comment by tzach** [comment]
Thanks for reporting!
Note that in 6.2 JMX is optional.
Can you please explain what you are using it for?

**Comment by samakovuluk** [comment]
any update on this?

**Comment by tzach** [comment]
@yaronkaikov do we have a .deb file for JMX, or is it only a Maven package?

**Comment by tzach** [comment]
@samakovuluk can you please elaborate on why do you need JMX to begin with?
It is no longer used for nodetool

**Comment by yaronkaikov** [comment]
> [@yaronkaikov](https://github.com/yaronkaikov) do we have a .deb file for JMX, or is it only a Maven package?

We don't build it anymore, although it can be constructed manually using `/reloc/build_reloc.sh` and then `/reloc/build_deb.sh`

**Comment by alsarins** [comment]
Same question here. We use a couple of JMX exporters with cassandra for monitoring on Ubuntu servers (22.04). It would be great to reuse them with scylla also. 

For now it's possible to build it with build_reloc.sh and build_deb.sh  as previous author suggested (although script build_deb.sh has an issue with tar.gz file name as it seems, but it can be fixed with renaming tar.gz file).  It will be great if we can just download deb package instead.

**Comment by avikivity** [comment]
I recommend to switch to monitoring via Prometheus. While the JMX package still exists, it's not maintained and is likely to bit-rot soon.

**Comment by alsarins** [comment]
Yes, we already use Prometheus exporter as well. But we hit the limit of Prometheus with number of metrics per host. That's why it's convenient to use JMX in addition to export additional metrics to external database. Sad if jmx is unsupported, it makes migration from Cassandra a little harder.

**Comment by avikivity** [comment]
You can query the metrics directly from the node (port 9180) or the REST API (this is what JMX does).
